src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs

```diff
@@ -4,7 +4,12 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
-
+namespace System.IdentityModel.Selectors {
+  public abstract partial class X509CertificateValidator {
+    protected X509CertificateValidator() { }
+    public abstract void Validate(System.Security.Cryptography.X509Certificates.X509Certificate2 certificate);
+  }
+}
```

Comment:
The rest of the class that isn't included are a single instance method used by the configuration system on desktop (so we wouldn't include that), and static properties or methods to provide validators with specific implementations (e.g. None). I believe those should have their current behavior evaluated and would possibly be superseded by a newer implementation with a different behavior as certificate validation criteria has moved on a bit since the code was originally written. At that point, we would likely have more classes we would want to add to a new IdentityModel contract and we would type forward this class, likely with no changes.